docs/UPGRADING.md

```diff
@@ -32,6 +32,31 @@ The `parseArch` parameter was removed since it was unused and was always true. T
 always parsed by the function. Furthermore the function now should also correctly interpret `packager:arch` spelled with
 the wrong casing.
 
+### `github.com/arduino/arduino-cli/i18n.Init(...)` now takes only one argument
+
+The signature of the function changed from:
+
+```go
+func Init(configLocale ...string) {
+```
+
+to
+
+```go
+func Init(configLocale string) {
+```
+
+If any extra argument was passed it can be safely removed since it was previously ignored.
```

Comment:
````suggestion
### `github.com/arduino/arduino-cli/i18n.Init(...)` now requires an empty string to be passed for autodetection of locale

For automated detection of locale, change the call from:

```go
func Init() {
```

to

```go
func Init("") {
```
````
Previously, only 0 or 1 arguments were accepted. It never allowed extra arguments.